World/Environment/Lightning.gdshader

```diff
@@ -0,0 +1,5 @@
+shader_type canvas_item;
+
+void fragment() {
```

Comment:
this file seems unneeded?

Reply:
True. Removed via [b78ea31](https://github.com/boku-ilen/landscapelab/pull/326/commits/b78ea31193f6854d2ef8cccd2ee5810ed94f81fa)